### DISCUSSION TITLE
Creating a Quarkus extension for AWS CloudWatch

### DISCUSSION THREAD

**Discussion by giscus** [quarkusio/quarkus#22867] [Quarkus Blog/Website]
# Creating a Quarkus extension for AWS CloudWatch

Quarkus: Supersonic Subatomic Java

https://quarkus.io/blog/quarkus-aws-cloudwatch_extension/

**Comment by chinmoydas1** [comment]
The extension is really helpful!! Is there any way to configure only using sts, instead of quarkus.log.cloudwatch.access-key-id, quarkus.log.cloudwatch.access-key-secret? I need to use it in EKS or Lambda and cannot provide these two properties there?

**Comment by chinmoydas1** [comment]
The extension is really helpful. Is there any way to configure using sts, instead of quarkus.log.cloudwatch.access-key-id, quarkus.log.cloudwatch.access-key-secret?
